Pipfile

```diff
@@ -7,6 +7,19 @@ name = "pypi"
 falcon = "~= 2.0.0"
 gunicorn = "~= 19.9"
 jsonschema = "~= 3.0"
+numpy = "*"
+scipy = "*"
+pandas = "*"
+sympy = "*"
+fuzzywuzzy = "*"
+python-dateutil = "*"
+pendulum = "*"
+arrow = "*"
+yarl = "*"
+pyyaml = "*"
+toml = "*"
```

Comment:
I'm not sold on including toml & yaml libraries since there are no write permissions nor internet access.